doc/modules/preprocessing.rst

```diff
@@ -610,6 +610,9 @@ a transformer that applies a log transformation in a pipeline, do::
     array([[ 0.        ,  0.69314718],
            [ 1.09861229,  1.38629436]])
 
+You can ensure that ``func`` and ``inverse_func`` are the inverse of each other
```

Comment:
Please note that a warning is raised and can be turned into an error with: give simplefilter example which tests message